test/units/galaxy/test_collection_install.py

```diff
@@ -226,17 +226,18 @@ def test_build_requirement_from_path_no_version(collection_artifact, monkeypatch
     mock_display = MagicMock()
     monkeypatch.setattr(Display, 'display', mock_display)
 
-    manifest_path = os.path.join(collection_artifact[0], b'MANIFEST.json')
```

Comment:
@s-hertel I think a negative test for `MANIFEST.json` would also be useful.

Reply:
Oh, true. Adding that now.